sql/moz-fx-data-shared-prod/search_terms_derived/suggest_impression_sanitized_v3/query.sql

```diff
@@ -56,7 +58,7 @@ sanitized_queries_count AS (
 -- output, which allows us to raise an error in the WHERE clause.
 validated_queries AS (
   SELECT
-    * EXCEPT (_n, _n_with_query),
+    * EXCEPT (_n, _n_with_query, region, country),
```

Comment:
I think these should be removed earlier.

```suggestion
    * EXCEPT (_n, _n_with_query),
```